tests/runtest.proj

```diff
@@ -412,6 +412,21 @@ namespace $([System.String]::Copy($(Category)).Replace(".","_").Replace("\","").
 
   </Target>
 
+  <Target Name="RunPerfTests" Condition="'$(Performance)'=='true'">  
+    <Message Text="Executing steps for perf tests" Importance="High"/>  
+    
+    <!-- generate project.json for runtime dependency -->  
+    <MSBuild Projects="$(MSBuildProjectFile)"  
+             Targets="CreateTestRuntimeJsonFile"/>  
+     
+    <!-- Package the product -->  
+    <MSBuild Projects="$(MSBuildProjectFile)" Targets="CreateTestOverlay" />  
```

Comment:
corerun.exe dependency is add in src\common\test_runtime\project.json
"Microsoft.NETCore.TestHost": "1.0.0-rc2-23816", 
Above package should bring in corerun.exe